.github/workflows/solidity-tests.yml

```diff
@@ -0,0 +1,45 @@
+name: Solidity Tests CI
+
+on:
```

Comment:
Make this manual. There will be a lot of tests so we don't want to run them always so it should be from a workflow kick off

Reply:
Done.